cmd/monaco/main.go

```diff
@@ -206,12 +208,22 @@ Examples:
   Deploy a specific project to a specific tenant:
     monaco deploy --environments environments.yaml --specific-environment dev --project myProject
 `
-	deployCommand := getDeployCommand(fs)
+	var deployCommand cli.Command
+
+	if isEnvFlagEnabled("NEW_CONFIG") {
```

Comment:
Shouldn't we make `NEW_CONFIG` a default? We should rather allow compatibility mode with `CONFIG_V1` flag, but use new config as default.

Reply:
As pointed out above, I would ignore this file and delete it.